policyengine_us/parameters/gov/states/mn/tax/income/subtractions/elderly_disabled/base_amount.yaml

```diff
@@ -6,10 +6,12 @@ metadata:
   breakdown:
   - filing_status
   reference:
-  - title: 2021 Minnesota Schedule M1R form and instructions
+  - title: 2021 Minnesota Schedule M1R form and instructions, Line 1
     href: https://www.revenue.state.mn.us/sites/default/files/2023-01/m1r_21.pdf#page=1
-  - title: 2022 Minnesota Schedule M1R form and instructions
+  - title: 2022 Minnesota Schedule M1R form and instructions, Line 1
     href: https://www.revenue.state.mn.us/sites/default/files/2023-01/m1r_22.pdf#page=1
+  - title: 2023 Minnesota Schedule M1R form and instructions, Line 1
+    href: https://www.revenue.state.mn.us/sites/default/files/2023-12/m1r-23.pdf#page=1
```

Comment:
could you add the legal code - it seems like it was missed in most of the parameter files when originally creating it?